src/Lwi/Client/ServiceDescription/Inside.php

```diff
@@ -77,6 +77,32 @@
 //            ),
 //            'responseClass' => 'AssetType',
 //        ),
+        'listProducts' => array(
+            'httpMethod'       => 'GET',
+            'uri'              => 'products',
+            'summary'          => 'List products',
+            'parameters'       => array(
+                'page' => array(
+                    '$ref' => 'PageParam',
+                ),
+                'page_size' => array(
+                    '$ref' => 'PageSizeParam',
+                ),
+                'query' => array(
```

Comment:
Is this param really implemented for products? As far as I know, the param is used only for listing assets.